### DISCUSSION TITLE
How to make the simulation-time synchronized with clock-time?

### DISCUSSION THREAD

**Discussion by TheASingh** [CURENT/andes#267] [Q&A] [answered]
Hi @cuihantao and @jinningwang ,

How can I synchronize the **simulation-time** with **clock-time**?

I found that increasing/decreasing time-step size impacts the simulation-time.
What would be the optimal way of synchronize both times?

Regards,
A Singh

**Comment by cuihantao** [comment, marked as answer]
There is an option `-qrt` for `andes run`. It is only a best-effort synchronization. As you decrease the step size, there will be more computation, which will make it naturally slower.

**Reply by TheASingh** [reply]
I tried supplying **qrt** and **kqrt**. But 20 seconds simulation is completing in 3.6606 clock-seconds.

Am I giving correct values?

![image](https://user-images.githubusercontent.com/96423656/153663812-7c4a2057-f15d-45b9-b61c-fe868376ad44.png)

**Reply by cuihantao** [reply]
You missed `config` in `ss.TDS.qrt`. Should be `ss.TDS.config.qrt`. Also, setting `kqrt` will make it 10 times slower than the real time.

**Reply by TheASingh** [reply]
Thanks, it solved my purpose.
